packages/threaddb/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import 'regenerator-runtime/runtime'
```

Comment:
In addition to moving these files, I've also auto-linted them to match our linter settings here.